superset/assets/stylesheets/less/variables.less

```diff
@@ -107,9 +107,21 @@
 /* Z-INDEX                                                              */
 /* Think of the site as "layers" rather than an arms race of numbers    */
 /* Try to keep these to a minimum                                       */
+/* Try to label semantic "layers" and add comments for usage notes      */
+/* Use double dash modifiers to step up/down from a base layer          */
+/* e.g. z-whatever--modifier                                            */
 /************************************************************************/
-@z-index-max: 1000;
-@z-index-dropdown: 11;
+
+/************************ toast messages, popovers **********************/
+@z-max: 3000;
+
+/***** filters, dashboard editor widgets, Explore reloading overlay *****/
+@z-dropdown: @z-above-dash-charts + 1;
+@z-above-dash-charts: 10;
```

Comment:
why not `dashboard` instead of `dash`?

Reply:
I was just trying to be terse... I'll make it dashboards, that's fine.